test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```

Comment:
- Just i have suggested in your .gitignore file, kindly add test.md to gitignore. 👍🏽